src/main/java/ch/njol/skript/expressions/ExprRepeated.java

```diff
@@ -0,0 +1,61 @@
+package ch.njol.skript.expressions;
+
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+@Name("Repeated")
+@Description("A string repeated a number times")
+@Examples({"on chat:", "\tset {_t} to message repeated 3 times", "\tsend {_t}"})
+@Since("INSERT VERSION")
+public class ExprRepeated extends SimpleExpression<String> {
+	static {
+		Skript.registerExpression(ExprRepeated.class, String.class, ExpressionType.COMBINED, "%string% repeat[ed] %number% time[s]", "repeat[ed] %string% %number% time[s]");
```

Comment:
The second pattern needs to have a test with variables, this kind of pattern tends to error out when you do something like `repeat {_string} {_number} times`.

Also, missing new line after ending brace (this is also the case in other places).

Reply:
Just tested, it works fine with variables